data/formations_basic.txt

```diff
@@ -51,7 +51,6 @@ formation "Vic"
 
 
 # X shaped formation that rotates with the flagship.
-# For warship-sized escorts
```

Comment:
I couldn't find the scaling code in this PR. Maybe it got lost in the merge conflicts? If it's not planned for the PR, the comment should stay.

Reply:
The numbers below (spacing of 150) suggest to me that this formation is aimed at interceptor-sized escorts. So I removed this comment, since the spacing seems to small for warships.

Reply:
Scaling (using the MultiAxisPoint class) got taken out of this PR, because it was too complex.
I filed an issue (https://github.com/endless-sky/endless-sky/issues/10324) for a simpler, alternative, implementation.